src/core/os_glfw.h

```diff
@@ -10,6 +10,9 @@
 #include <GLFW/glfw3native.h>
 #endif
 
+#ifdef OS_GLFW_H
```

Comment:
I did something dubious here. Right now os_glfw.h can only be included in one file because it defines non-static symbols. But os_glfw seems(?) to be the only way we have of getting glfw.h. For this reason gamepad.c #defines OS_GLFW_H and then includes os_glfw.h. Maybe this could be done a cleaner way.

Reply:
`os_glfw` is designed to only be included in one translation unit right now, the `os_*.c` translation units (since it provides most of the symbols in os.h).

Joystick input could go through the existing "os" layer or GLFW could be included directly.

Reply:
If it's OK with you I think I'll switch to including GLFW directly.

I wonder if the driver should be named "GLFWGamepad" instead of "Gamepad", or something.